libraries/GCS_MAVLink/include/mavlink/v1.0/common/version.h

```diff
@@ -5,7 +5,7 @@
 #ifndef MAVLINK_VERSION_H
 #define MAVLINK_VERSION_H
 
-#define MAVLINK_BUILD_DATE "Wed Jul 20 13:14:23 2016"
+#define MAVLINK_BUILD_DATE "Thu Jul 21 14:44:48 2016"
```

Comment:
Is this mavlink rebuild compatible with the work that Benoit has been doing with syncing mavlink versions across the board?